pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "ntc_rosetta"
-version = "0.2.0"
+version = "0.2.2"
```

Comment:
0.2.1 was published manually at some point and time on someone's local machine but I check it and there is no code diff between what is in this repo and what was in PyPI